### ISSUE TITLE
Incorporate instructions for running the SESANS GUI (Trac #720)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#842]
Add-in Wim's instructions for running the SESANS GUI to sasmodels and link in sasview.

Migrated from http://trac.sasview.org/ticket/720
```json
{
    "status": "closed",
    "changetime": "2016-10-07T16:32:36",
    "_ts": "2016-10-07 16:32:36.246790+00:00",
    "description": "Add-in Wim's instructions for running the SESANS GUI to sasmodels and link in sasview.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-10-07T13:12:39",
    "component": "SasView",
    "summary": "Incorporate instructions for running the SESANS GUI",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "enhancement"
}
```

**Comment by smk78** [comment]
Trac update at `2016/10/07 15:29:22`: **smk78** commented:

In changeset c34a31f9923326734c823a9bf9c47652ff0475e7:

```text
#!CommitTicketReference repository="sasmodels" revision="c34a31f9923326734c823a9bf9c47652ff0475e7"
Added SESANS fitting instructions to sasmodels. Refs #842
```

**Comment by smk78** [comment]
Trac update at `2016/10/07 16:32:36`: 
* **smk78** commented:

In changeset 751f4bdb1383a4267910b9a189f51206749158e0:

```text
#!CommitTicketReference repository="sasview" revision="751f4bdb1383a4267910b9a189f51206749158e0"
SESANS fitting instructions now linked in sasview also. Closes #842
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
